system/HTTP/Response.php

```diff
@@ -188,6 +188,7 @@ public function __construct($config)
 
     /**
      * Turns "pretend" mode on or off to aid in testing.
+     *
      * Note that this is not a part of the interface so
      * should not be relied on outside of internal testing.
```

Comment:
Let's start annotating these `@testTag` to indicate the class/method is only for testing (following this convention: https://github.com/DaveLiddament/php-language-extensions#testtag). Once we have Attributes we can decide if we want to elevate these to an actual attribute - but at least we can start accumulating them now.

I could probably craft a custom PHPStan rule to work with the annotation in the meantime as well.

Reply:
Added `@testTa`